tools/common.py

```diff
@@ -0,0 +1,10 @@
+def include_dir_order():
```

Comment:
Nit: This is the third example of function name capitalization style in this set of related files.